src/NServiceBus.Core/Sagas/Saga.cs

```diff
@@ -74,7 +76,7 @@ protected void RequestTimeout<TTimeoutMessageType>(DateTime at, Action<TTimeoutM
         /// </summary>
         /// <param name="at"><see cref="DateTime"/> to send timeout <paramref name="timeoutMessage"/>.</param>
         /// <param name="timeoutMessage">The message to send after <paramref name="at"/> is reached.</param>
-        protected void RequestTimeout<TTimeoutMessageType>(DateTime at, TTimeoutMessageType timeoutMessage)
+        protected Task RequestTimeout<TTimeoutMessageType>(DateTime at, TTimeoutMessageType timeoutMessage)
```

Comment:
Why in some methods (eg. `SendAsync`) you have `Async` suffix and in this method you omit that? It's a bit inconsistent.